prow/spyglass/spyglasstests/podlogartifact_test.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package spyglasstests
+
+import (
+	"bytes"
+	"testing"
+
+	"k8s.io/test-infra/prow/spyglass"
+)
+
+func TestPodLogReadAll(t *testing.T) {
+	testCases := []struct {
+		name     string
+		artifact *spyglass.PodLogArtifact
+		expected []byte
+	}{
+		{
+			name:     "\"Job\" Podlog readall",
+			artifact: spyglass.NewPodLogArtifact("job", "123", "", 500e6, fakeJa),
+			expected: []byte("clusterA"),
```

Comment:
eh why this gives back `clusterA`? :-)

Reply:
this is taken directly from the original test written for the podlog endpoint in `deck/jobs/job_test.go` test `TestGetLog`